com.github.benini.scid.yml

```
@@ -0,0 +1,122 @@
+app-id: com.github.benini.scid
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: /app/scid/scid
+
+finish-args:
+  - --filesystem=home        # for accessing files in the user's home folder
+  - --share=network          # access the network
+  - --socket=x11             # show windows with x11
+  - --share=ipc              # share IPC namespace with the host (necessary for X11)
+  - --device=dri             # OpenGL rendering
+  - --socket=pulseaudio      # play sound with PulseAudio
+
+modules:
+- name: stockfish # Stockfish engine, put into /app/engines/stockfish/
+  buildsystem: simple
+  build-commands:
+  - cp nn-ad9b42354671.nnue ./src
+  - cd src; make -j build
+  - mkdir -p /app/engines/stockfish
+  - cp ./src/stockfish /app/engines/stockfish/
+  - cp ./nn-ad9b42354671.nnue /app/engines/stockfish/
+  sources:
+  - type: archive
+    url: https://github.com/official-stockfish/Stockfish/archive/refs/tags/sf_15.1.zip
+    sha256: 5174e4247f4c107648c9f7c906d5e9733abc61c6dae047be570cee2b930f0339
+  - type: file
+    url: https://tests.stockfishchess.org/api/nn/nn-ad9b42354671.nnue
+    sha256: ad9b423546714137916bd38978af6fd68d7b8951bef25ff76bf43da72d6cb786
+
+- name: eigen # library for linear algebra, used for lc0
+  buildsystem: cmake-ninja
+  builddir: true
+  sources:
+  - type: archive
+    url: https://gitlab.com/libeigen/eigen/-/archive/3.4.0/eigen-3.4.0.tar.gz
+    sha256: 8586084f71f9bde545ee7fa6d00288b264a2b7ac3607b974e54d13e7162c1c72
+
+- name: openblas # Basic Linear Algebra Subprograms library, used for lc0
+  no-autogen: true
+  make-args:
+  - DYNAMIC_ARCH=1
+  - FC=gfortran
+  - NO_LAPACKE=1
+  - USE_OPENMP=0
+  make-install-args:
+  - PREFIX=/app
+  post-install:
+  - rm /app/lib/*.a
```

```suggestion
  post-install:
  - rm /app/lib/*.a
```

I recommend using `cleanup` (per module or top-level) for things like these.